pm/sendermonitor_test.go

```diff
@@ -470,7 +470,7 @@ func TestReserveAlloc(t *testing.T) {
 	assert.Zero(expectedAlloc.Cmp(alloc))
 }
 
-func TestValidateSender(t *testing.T) {
+func TestValidateSender_SenderMonitor(t *testing.T) {
```

Comment:
`<TYPE_NAME>`_`<METHOD_NAME>` feels a bit clearer to me as it gets more specific about what is under test (type under test followed by method under test) as you continue reading the test name. WDYT?

Reply:
fixed in 3b14e74